dazzle-rewrite/chunks/lang-node/chunk.yaml

```diff
@@ -0,0 +1,10 @@
+variants:
+  - name: "16.13.0"
+    args:
+      NODE_VERSION: 16.13.0
+  - name: "16.13.1"
+    args:
+      NODE_VERSION: 16.13.1
+  - name: "17"
```

Comment:
What are the downside of using hashes with chunked image? Is it the build time or users needing to pull a new chunk because a new hash will be generated when rebuilding?